xml/x509_subjects.xml

```diff
@@ -168,6 +168,14 @@
     <param pos="0" name="os.product" value="iDRAC Linux"/>
   </fingerprint>
 
+  <fingerprint pattern="^CN=opensnort\.example.com,O=Sourcefire,emailaddress=support@sourcefire.com">
+    <description>Sourcefire Snort HTTPS console w/Service tag</description>
```

Comment:
`w/Service tag` - Is that something in the fingerprint or context that didn't get copied here?